Components/Images/Images.html

```diff
@@ -0,0 +1,103 @@
+                    <li class="sidebar_submenu_list"><a href="../Badge/badge.html">Badge on Avators</a></li>
+                </ul>
+                <li class="sidebar_menu_list"><a href="../Button/button.html">Button</a></li>
+                <ul class="sidebar_submenu">
+                    <li class="sidebar_submenu_list"><a href="../Button/button.html">Primary</a></li>
+                    <li class="sidebar_submenu_list"><a href="../Button/button.html">Link</a></li>
+                    <li class="sidebar_submenu_list"><a href="../Button/button.html">Icon Button</a></li>
+                    <li class="sidebar_submenu_list"><a href="../Button/button.html">Floating Action Button</a></li>
+                </ul>
+                <li class="sidebar_menu_list"><a href="../Card/card.html">Card</a></li>
+                <ul class="sidebar_submenu">
+                    <li class="sidebar_submenu_list"><a href="../Card/card.html">Cards with badges</a></li>
+                    <li class="sidebar_submenu_list"><a href="../Card/card.html">Cards with dismiss</a></li>
+                    <li class="sidebar_submenu_list"><a href="../Card/card.html">Cards with text overlay</a></li>
+                    <li class="sidebar_submenu_list"><a href="../Card/card.html">Text only cards</a></li>
+                    <li class="sidebar_submenu_list"><a href="../Card/card.html">Vertical/Horizontal cards</a></li>
+                    <li class="sidebar_submenu_list"><a href="../Card/card.html">Cards with shadow</a></li>
+                </ul>
+                <li class="sidebar_menu_list"><a href="#">Image</a></li>
+                <ul class="sidebar_submenu">
+                    <li class="sidebar_submenu_list"><a href="#">Responsive Images</a></li>
+                    <li class="sidebar_submenu_list"><a href="#">Round Image</a></li>
+                </ul>
+                <li class="sidebar_menu_list"><a href="../Input/input.html">Input</a></li>
+                <ul class="sidebar_submenu">
+                    <li class="sidebar_submenu_list"><a href="../Input/input.html">Textbox</a></li>
+                    <li class="sidebar_submenu_list"><a href="../Input/input.html">With validation and error styles</a></li>
+                </ul>
+            </ul>
+        </aside>
+        <!-- Sidebar Ends -->
+        <div class="container">
+            <h1 class="container_header">Image</h1>
+            <p class="container_header_content">Imagery in UI design is the visual language used to help users relate to
+                their brand or product. </p>
+            <hr>
+            <h2 class="container_subheader">Responsive Image</h2>
+            <p class="container_subheader_content">Responsive Image Component are the screen responsive in which we set
+                the width to 100% and height to auto.
+            </p>
+            <div class="image_responsive">
+                <img class="image_wrapper" src="/image/landscape.jpg" alt="landscape">
```

Comment:
This image is taking a lot of time to load because of its larger size. Maybe try a lighter version

Reply:
will look for later on after completing it